atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -132,12 +122,7 @@ static void waitForSchemaVersions(
         throw new IllegalStateException(errorMessage);
     }
 
-    /**
-     * @param allowQuorumAgreement if true, requires only a quorum of nodes to be in agreement. If false, all nodes
-     * need to have the same schema version.
-     */
-    private static boolean requiredNumberNodesAgreeOnSchemaVersion(
-            boolean allowQuorumAgreement,
+    private static boolean quorumOfNodesAgreesAndOthersUnreachable(
```

Comment:
thoughts on renaming to `quorumOfNodesAvailableAndAllAvailableNodesAgree`? Seems more directly in line with the logic we're going for here.

Reply:
or flip the ordering `allAvailableNodesAgreeAndQuorumOfNodesAvailable` (which is the ordering of the impl of the method in practice).